examples/flows/standard/conditional-flow-for-if-else/README.md

```diff
@@ -0,0 +1,47 @@
+# Conditional flow for if-else scenario
+
+This example is a conditonal flow for if-else scenario.
+
+In this flow, it checks if an input query passes content safety check. If it's denied, we'll return a default response; otherwise, we'll call LLM to get a response and then summarize the final results.
+
+:::{admonition} Notice
+The 'content_safety_check' and 'llm_result' node in this flow are dummy nodes that do not actually use connections of `Azure Content Safty` and `Azure Open AI`. You can replace them with the real ones. Learn more: [Manage connections](../../../connections/connection.ipynb)
```

Comment:
do not actually use the content safety tool and LLM tool?

Reply:
Updated.